src/MSBuildTaskHost/OutOfProcTaskHost.cs

```diff
@@ -88,13 +88,20 @@ public static int Main()
         /// </returns>
         internal static ExitType Execute()
         {
-#if FEATURE_DEBUG_LAUNCH
-            // Provide Hook for debugger
-            if (Environment.GetEnvironmentVariable("MSBUILDDEBUGONSTART") == "1")
+            switch (Environment.GetEnvironmentVariable("MSBUILDDEBUGONSTART"))
             {
-                Debugger.Launch();
-            }
+#if FEATURE_DEBUG_LAUNCH
+                case "1":
+                    Debugger.Launch();
+                    break;
 #endif
+                case "2":
+                    // Sometimes easier to attach rather than deal with JIT prompt
+                    Process currentProcess = Process.GetCurrentProcess();
+                    Console.WriteLine($"Waiting for debugger to attach ({currentProcess.MainModule.FileName} PID {currentProcess.Id}).  Press enter to continue...");
```

Comment:
I think this would be right because of the ()?
```
Waiting for debugger to attach (d:\src\msbuild.fork\artifacts\Debug\bootstrap\net46\MSBuild\15.0\Bin\MSBuild.exe PID 14408).  Press enter to continue...
```